install.sh

```diff
@@ -20,9 +20,25 @@
 install_path=/usr/local/bin
 slacktee_script="slacktee.sh"
 
+skip_setup=false
+
+# Parse options
 if [[ $# -ne 0 ]]; then
-    install_path=$1
+	while [[ $# -gt 0 ]]; do
+		opt="$1"
+		shift
+
+		case "$opt" in
+			-s|--skip-setup)
+				skip_setup=true
+				;;
+            *)
```

Comment:
Super nitpick, but the spacing on these case statements looks a little weird.

Reply:
Ohh, thank you for catching it! Fixed it!